R/modify_pd.R

```diff
@@ -0,0 +1,53 @@
+#' Update indention information of parse data
+#'
+#' @param pd A nested or flat parse table that is already enhanced with
+#'   line break and space information via [create_filler()] or
+#'   [create_filler_nested()].
+#' @param indent_by How many spaces should be added after the token of interest.
+#' @name update_indention
+NULL
+
+#' @rdname update_indention
+indent_round <- function(pd, indent_by) {
+  if (any(pd$token == "')'")) {
+    start <- 2
```

Comment:
Please check boundaries: Do we really need to indent the second token, or perhaps only starting from the third?